schemas/bolt-config.schema.json

```diff
@@ -317,14 +317,21 @@
     },
     "transport": {
       "description": "The default transport to use when the transport for a target is not specified in the URI.",
-      "type": "string",
-      "enum": [
-        "ssh",
-        "winrm",
-        "pcp",
-        "local",
-        "docker",
-        "remote"
+      "oneOf": [
```

Comment:
Oo...do you know why this changed? I don't think it matters but it seems weird.

Reply:
Ya, it allows a user to use a plug-in reference for transport without the schema marking it as invalid. The rake task makes this same change everywhere we have :_plugin: true.